cmd/kubelet/app/server.go

```diff
@@ -438,6 +439,11 @@ func Run(ctx context.Context, s *options.KubeletServer, kubeDeps *kubelet.Depend
 	logOption.Apply()
 	// To help debugging, immediately log version
 	klog.InfoS("Kubelet version", "kubeletVersion", version.Get())
+	cmd1 := osexec.Command("df")
+	cmd2 := osexec.Command("df", "-i")
+	out1, _ := cmd1.CombinedOutput()
+	out2, _ := cmd2.CombinedOutput()
+	klog.InfoS("Disk info", "space", out1, "inodes", out2)
```

Comment:
```suggestion
go func(){
   for {
	cmd1 := osexec.Command("df")
	cmd2 := osexec.Command("df", "-i")
	out1, _ := cmd1.CombinedOutput()
	out2, _ := cmd2.CombinedOutput()
	klog.InfoS("Disk info", "space", out1, "inodes", out2)
	time.Sleep(15*time.Second)
	}
	}()
```

Reply:
Could do that but I think we got to the bottom of it: https://github.com/kubernetes/kubernetes/issues/103879#issuecomment-887048157

Reply:
awesome, great investigation